docs/Tutorials/demo/budget_with_farming.md

```diff
@@ -11,33 +11,33 @@ The [budget](https://github.com/tendermint/budget) and [farming](https://github.
 
 ## What does budget module do?
 
-A budget module is a Cosmos SDK module that implements budget functionality. It is an independent module from other SDK modules and core functionality is to enable anyone to create a budget plan through governance param change proposal. Once it is agreed within the community, voted, and passed, it uses the budget source address to distribute amount of coins by the rate defined in the plan to the collection address. Collecting all budgets and distribution take place every epoch blocks that can be modified by a governance proposal.
+The budget module is a Cosmos SDK module that implements budget functionality. It is an independent module from other SDK modules and core functionality is to enable anyone to create a budget plan through governance param change proposal. After it is agreed within the community, voted, and passed, it uses the budget source address to distribute amount of coins by the rate that is defined in the plan to the destination address. Collecting all budgets and distribution take place every epoch blocks that can be modified by a governance proposal.
 
-One use case is for Gravity DEX farming plan. Budget module can be used to create a budget plan that defines Cosmos Hub's FeeCollector module account where transaction gas fees and part of ATOM inflation are collected as budget source address and uses custom module account (created by budget creator) as collection address. 
+One use case is the Gravity DEX farming plan. Budget module can be used to create a budget plan that defines the Cosmos Hub FeeCollector module account where transaction gas fees and part of ATOM inflation are collected as budget source address and uses a custom module account (created by budget creator) as the collection address. 
 
-## What does farming module do?
+## What does the farming module do?
 
-A farming module is a Cosmos SDK based module that implements farming functionality that provides farming rewards to staking participants called farmers. 
+The farming module is an Cosmos SDK-based module that implements farming functionality that provides farming rewards to staking participants called farmers. 
```

Comment:
Should we use **an** before **Cosmos**?

Reply:
@hallazzang using "a" is the correct article here, because Cosmos is a hard consonant

let me try a rewrite for clarity

Reply:
```suggestion
The farming module is a Cosmos SDK-based module that implements farming functionality that provides farming rewards to staking participants called farmers. 
```

Reply:
It looks like the suggested change hasn't been applied. I will apply it and approve the PR.